Detectors/FIT/FT0/simulation/src/Digitizer.cxx

```diff
@@ -25,22 +25,22 @@ using namespace o2::ft0;
 
 ClassImp(Digitizer);
 
-auto Digitizer::get_time(const std::vector<double>& times, double deadTime) -> CFDOutput
+Digitizer::CFDOutput Digitizer::get_time(const std::vector<double>& times, double deadTime)
```

Comment:
I think this function should be common for FT0, FV0 and FDD. Perhaps it could be moved to a separate class (and file), that is available for all 3 FIT detectors. The other anticipated functions, like the one that takes care of the dead time of the charge integrator, could be placed into the same common class later.

Reply:
I think we should discuss this on our FIT software meeting. Now lets keep it for FT0